.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Is this a required file or just for convenience?

Reply:
Alright, yeah I'd rather not have extra files like that as it's up to the personal workflow if it is needed.